content/release/latest-stable-release-sabayon-1804.md

```diff
@@ -0,0 +1,87 @@
++++
+title = "Sabayon 18.04 - Finally a new stable release"
+description = "18.04 Press release"
+date = "2018-04-09"
+categories = ["releases", "sabayon"]
+tags = ["releases", "sabayon"]
++++
+
+The team behind Sabayon is excited to present you the latest release:
+Sabayon 18.04.
+
+Sabayon is a modern and easy to use distribution based on Gentoo,
+which follows a reliable rolling release model.
+
+Originally we planned to release it last December,
+but then an update with GCC on Gentoo profiles made us rebuild almost all
+packages.
+
+Please read on or [download](/download/) your flavour :-)
+
+In general, the release is a new snapshot so that after fresh install you
+have less updates, but there is more: it has been worked out, tested, improved,
+fixed and discussed, all within the small but active team of voluteers that
+work on this distribution, so that you can get it, install it and enjoy it!
+
+## Changes
+
+* Kernel 4.14
+* GNOME 3.24
+* KDE 5.12
+* Mate 1.18
+* Xfce 4.12
+
+### Some picks
+
+Sabayon has now a [Facebook profile](https://www.facebook.com/sabayon.linux)
+with a speaking name next to our
+[Facebook group](https://www.facebook.com/groups/36125411841).
+
+Development is open, so if you want to have a look at what has been done in
+detail, it's right there; you can find the link in the Download section below.
+
+### What was dropped?
+
+With this release, we are in the process of dropping support for Qt4, which
+[reached its end of life in december 2015](https://wiki.qt.io/Main#Quick_Access_.28Portal.29) (!)
+
+Due to changes in its policy, we had to take down or Vagrant image.
+We still have a [Docker profile](https://hub.docker.com/r/sabayon) though.
+
+### Plans for the future
+
+We are already planning what to do for the next stable release (hopefully to be released this year!)
+For example, we would like to stick to one DisplayManager greeter across all our flavours.
+Or drop Anaconda as system installer and use Calamares.
```

Comment:
I read it as "we'll change the display manager OR we'll change the installer."
Second, `DisplayManager` -> `display manager`.

Reply:
Fixing myself.